### ISSUE TITLE
Revising Metric: Time Inclusion for Virtual Events

### ISSUE THREAD

**Issue by germonprez** [chaoss/wg-dei#433]
### Name of the Metric Being Revised
Time Inclusion for Virtual Events
https://chaoss.community/metric-time-inclusion-for-virtual-events/

### Link to Original Metric Release Issue
https://github.com/chaoss/wg-dei/issues/370

### Specific Metric Details that Require Attention

- 'Examples of Time Inclusion' should be 'Examples of Time Inclusion for Virtual Events' 
- How is 'Network bandwidth' part of the metric? 
- How is 'low bandwidth' part of the metric? 
- Update the Likert Scale from [1-5] to [1-x]
- For survey of speakers, why is the second point important? 
- Update the references to the actual title
- Add date of last review 

#### Process

- [ ] Create the “revising metric” labeled issue in the authoring WG’s repo for comments during review period
- [ ] Create pull request of revised metric to WG’s repo (after checking Content Quality and Technical Requirements below)
- [ ] Add the metric revision to release notes issue in working group repo
- [ ] Update the [Metrics Spreadsheet](https://docs.google.com/spreadsheets/d/1tAGzUiZ9jdORKCnoDQJkOU8tQsZDCZVjcWqXYOSAFmE/edit) to indicate that the revised metric is under review
- [ ] Create issue in [CHAOSS/Translations repository](https://github.com/chaoss/translations) to kick-off translation to other languages (please use the the translation issue template)
- [ ] "Metric Candidate Release" label added to the metric release candidate issue when fully ready fo release

**When above steps are completed:**

- [ ] Announce new/updated metric on mailing list, newsletter, community Zoom call, and Twitter. This can be coordinated with the community manager and can be done as a collective of all revised metrics from the CHAOSS community. 

#### Content Quality (check all that apply)

- [ ] Date of last review has been added to the revised metric at the bottom of the markdown file (month/year)
- [ ] Formatting changes have been made 
- [ ] Minor editorial changes have been made 
- [ ] Major editorial changes have been made 
- [ ] Metric adheres to the current [metrics template](https://github.com/chaoss/community/blob/main/templates/metric-template.md)
- [ ] Metric name has changed and necessary updates are made to the WG README table and metrics tracking spreadsheet

#### Technical Requirements for any Revisions

- [ ] Metric file name is the full metric name and only contains lower case letters and hyphens (“-”) for spaces
- [ ] Images are included using markdown and absolute path links (as described in the metrics template)
- [ ] Images have at least one empty line above and below them
- [ ] Images are placed in image folder and followed [naming convention](https://github.com/chaoss/community/blob/main/templates/metric-template.md)
- [ ] If new a focus area is created, ensure focus area is added to wg repo readme and focus area folder readme
- [ ] There is no HTML code in the metrics markdown file

**Comment by pratik2315** [comment]
Hey @germonprez ! Do you think that this issue is okay to take for newcomers to the chaoss community?

**Comment by germonprez** [comment]
Yes! I would suggest that you make suggested changes via a PR to address some or all of the notes in this Issue.

**Comment by pratik2315** [comment]
Thanks. Ill try my best :)

**Comment by ElizabethN** [comment]
Here is where the PR should go: https://github.com/chaoss/wg-dei/blob/main/focus-areas/event-diversity/time-inclusion.md 
Also, can you change the name of the file to match the name of the metric? (time-inclusion-for-virtual-events.md)

**Comment by TejasMate** [comment]
Issues solved:
'Examples of Time Inclusion' should be 'Examples of Time Inclusion for Virtual Events'
Update the Likert Scale from [1-5] to [1-x]
Update the references to the actual title
Add date of last review
change file name

Issues unsolved:
How is 'Network bandwidth' part of the metric?
How is 'low bandwidth' part of the metric?
For survey of speakers, why is the second point important?
